### ISSUE TITLE
Address pending issues from PR #327

### ISSUE THREAD

**Issue by rranjan3** [hyperledger/avalon#335]
Address comments from PR #327
#331

**Comment by danintel** [comment]
The issues are (or include):
docs/dev-environments/ethereum/besu/docker-compose-truffle.yaml:

```
> @@ -24,7 +24,7 @@ services:
         - ./my_project:/my_project
         - $TCF_HOME/sdk/avalon_sdk/ethereum/contracts/proxy-model:/contracts
         - $TCF_HOME/sdk/avalon_sdk/ethereum/truffle_artifacts:/truffle_artifacts
```
Can these files be made part of container image by COPY in Dockerfile rather than volume mount. Though volume mount is helpful, it requires user to have a copy of Avalon code base locally. Going forward we also want to support K8s for Avalon and at that point it's not feasible to assume that Avalon code base will be downloaded in every K8s node.
________________________________________
docs/dev-environments/ethereum/besu/docker-compose-truffle.yaml:

```
> @@ -24,7 +24,7 @@ services:
         - ./my_project:/my_project
         - $TCF_HOME/sdk/avalon_sdk/ethereum/contracts/proxy-model:/contracts
         - $TCF_HOME/sdk/avalon_sdk/ethereum/truffle_artifacts:/truffle_artifacts
-      container_name: truffle-envt-setup
+      container_name: truffle-envt-besu-setup
       command: "bash -c \"\
           cd my_project && \
           truffle init && \
```

Can we move these commands to scripts and execute it. Also explore the options of running these inside container
________________________________________
docs/dev-environments/ethereum/ganache/cleanup.sh:

```
> +    if [ $? -ne 0 ]
+    then
+        echo "Script failed."
+        exit 1
+    else
+        echo "Done."
+    fi
+}
+
+echo ""
+echo "=================================================="
+echo "STEP 1 :: Stop Ganache cli and destroy container"
+echo "=================================================="
+docker stop local-ganache
+docker rm local-ganache
+check_and_exit
```

You could do something similar to error_exit() in https://github.com/hyperledger/avalon/blob/master/tools/rebuild.sh

for eg: sudo rm -rf caches database DATABASE_METADATA.json || error_exit "Failed to delete data in besu node1"

```
function error_exit()
{
    echo "[Error]: " ${*}
    exit 1
}
```

**Comment by rranjan3** [comment]
Fixed in #343.
